src/Error/Warning.php

```diff
@@ -14,6 +14,7 @@
  * Also, it is possible to override warning handler (which is **trigger_error()** by default).
  *
  * @phpstan-type WarningHandler callable(string $errorMessage, int $warningId, ?int $messageLevel): void
+ * @psalm-type WarningHandler callable(string, int, int|null): void
```

Comment:
Let's temporarily keep these so we suppress psalm errors in order to find other errors

https://github.com/vimeo/psalm/issues/7527

Reply:
Please link the relevant issue in a TODO above this line so we know when to remove it again. Same with other such workarounds.